CreateMessage/__tests__/handler.test.ts

```diff
@@ -538,12 +588,52 @@ describe("CreateMessageHandler", () => {
         content: {
           markdown: "md",
           subject: "subject",
-          third_party_data: { has_attachments: false, has_remote_content: false }
+          third_party_data: {
+            has_attachments: false,
+            configuration_id: anOtherRCConfiguration.configuration_id
+          }
         }
       } as ApiNewMessageWithDefaults,
       some(anotherFiscalCode)
     );
 
-    expect(r.kind).toBe("IResponseSuccessRedirectToResource");
+    expect(r.kind).toBe("IResponseErrorForbiddenNotYourConfiguration");
+  });
+
+  it("should return 403 ok if a configuration_id is not provided", async () => {
+    const mockGenerateObjId = jest
+      .fn()
```

Comment:
Can we add some tests in case the `mockMessagesServicesApiClient` return `400` `404` and `500`?

Reply:
https://github.com/pagopa/io-functions-services/pull/300/commits/c2033fe4ab849188ef9ddcab44f07affb46826a8